packages/lab/stories/radio-button/radio-button.stories.tsx

```diff
@@ -1,156 +1,191 @@
-      <DensityExample name="light" />
-    </SaltProvider>
-    <SaltProvider mode="dark">
-      <DensityExample name="dark" />
-    </SaltProvider>
-  </StoryScroller>
-);
+      <RadioButton label="Error unchecked" value="Error-unchecked" error />
+      <RadioButton label="Error checked" value="Error-checked" checked error />
+      <RadioButton
+        label="Disabled error"
+        value="disabled-error"
+        disabled
+        error
+      />
+      <RadioButton
+        label="Disabled error checked"
+        value="disabled-error-checked"
+        checked
+        disabled
+        error
+      />
+    </>
+  );
+};
 
-export const VerticalGroup = ({
-  title,
-  density,
-  name,
-}: ExampleWithTitleProps) => (
-  <Panel>
-    <SaltProvider density={density}>
-      <RadioButtonGroup
-        aria-label="Uncontrolled Example"
-        defaultValue="forward"
-        legend={"Legend"}
-        name={"Name"}
-      >
-        <RadioButton key="spot" label="Spot" value="spot" />
-        <RadioButton key="forward" label="Forward" value="forward" />
-        <RadioButton
-          disabled
-          key="option"
-          label="Option (disabled)"
-          value="option"
-        />
-      </RadioButtonGroup>
-    </SaltProvider>
-  </Panel>
+export const VerticalRadioButtonGroup = () => (
+  <SaltProvider>
+    <RadioButtonGroup>
+      <RadioButton key="option1" label="Radio Option 1" value="option1" />
+      <RadioButton key="option2" label="Radio Option 2" value="option2" />
+      <RadioButton
+        disabled
+        key="option3"
+        label="Radio Option 3 (disabled)"
+        value="option3"
+      />
+    </RadioButtonGroup>
+  </SaltProvider>
 );
 
-export const RowGroup = ({ title, density, name }: ExampleWithTitleProps) => (
-  <Panel>
-    <SaltProvider density={density}>
-      <RadioButtonGroup legend={title} name={name} row>
-        <RadioButton key="spot" label="Spot" value="spot" />
-        <RadioButton key="forward" label="Forward" value="forward" />
-      </RadioButtonGroup>
-    </SaltProvider>
-  </Panel>
+export const HorizontalRadioButtonGroup = ({
+  labelWrap,
+}: RadioButtonGroupProps) => (
+  <SaltProvider>
+    <RadioButtonGroup direction={"horizontal"} labelWrap={labelWrap}>
+      <RadioButton key="option1" label="Radio Option 1" value="option1" />
+      <RadioButton key="option2" label="Radio Option 2" value="option2" />
+      <RadioButton
+        disabled
+        key="option3"
+        label="Radio Option 3 (disabled)"
+        value="option3"
+      />
+    </RadioButtonGroup>
+  </SaltProvider>
 );
 
 interface DensityExampleProps {
+  density?: "high" | "medium" | "low" | "touch";
+  direction: "vertical" | "horizontal";
+  name: string;
+  title: string;
+}
+interface DensityExamplesProps {
   name: string;
 }
 
-const DensityExample = ({ name }: DensityExampleProps) => (
+const DensityExample = ({ density, direction, name }: DensityExampleProps) => (
```

Comment:
I'm wondering if we still need the Density examples. My initial feeling is that we don't

Reply:
I would expect them to be in our QA story. The same as the All example below